packages/metro-transformer/src/expo/index.ts

```diff
@@ -0,0 +1,5 @@
+import { createLinguiMetroTransformer } from "../metroTransformer"
+
+const expoTransformer = require("@expo/metro-config/babel-transformer")
+
+export const transform = createLinguiMetroTransformer(expoTransformer)
```

Comment:
no, changed to import :)

Reply:
actually, changed it back because rollup was complaining about `implicit dependency`. 

`"@expo/metro-config/babel-transformer"` is an optional peer dependency.

Reply:
If it's a dependency it should be explicitly stated in the one of the dependencies in package.json, i see only `metro` stated as peer dependency, but not a `@expo/metro-config` 

That might cause bugs due to package managers will be not aware that this package is using that dependency.

Also, what you mean by "optional peer dependency"?   I don't see any branching here, for a case where this dependecny would not be resolved, as well as [peerDependeciesMeta](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#peerdependenciesmeta)

Reply:
done

Reply:
Thanks, and now import could be used and rollup will not complain